panc/src/main/scripts/panlint/panlint.py

```diff
@@ -36,6 +36,7 @@
 RE_COMMENT_LINE = re.compile(r'^\s*' + RS_COMMENT + '.*$')
 RE_ANNOTATION = re.compile(r'@\w*{.*?}', re.S)
 RE_OPERATOR = re.compile(r'([>=<!?]=|[+*=/-])')
+RE_HEREDOC = re.compile(r'<<(\w+);\s*$.*?\1$', re.S|re.M)
```

Comment:
pep8 prefers:
```
RE_HEREDOC = re.compile(r'<<(\w+);\s*$.*?\1$', re.S | re.M)
```

which seems to be borne out by https://docs.python.org/3/howto/regex.html#compilation-flags